public/index.html

```diff
@@ -83,6 +83,49 @@ <h2>Usage</h2>
   <h2>Jquery example</h2>
   <p>View the source of this webpage to see how you can call the greencheck api service by using jquery</p>
 
+  <h2>Batch API</h2>
```

Comment:
@mrchrisadams I know this runs on the python backend, but isn't it a bit weird this is not under api.tgwf.org? 

Can we put that under api.tgwf.org/batch and do some vhost magic that nginx pushes that to the admin code handlers?

Reply:
Yeah, I know what you're saying - I don't think I know how I'd do that nginx magic, and I don't know enough about symfony to do redirects - is this something you're able to add?

If we had a v3 namespace like api.tgwf.org/v3/ and proxies requests over to admin.tgwf/api/v3/, I think that would largely work, and they're on the same machine, so I can't see it slowing stuff down too much.

We'd just need to take into account the idea thar uploaded lists of files might be very large, and keeping the response buffered  in memory in nginx, and then again in php and then again in python would be silly.